.github/workflows/codeql.yml

```
@@ -92,17 +91,6 @@ jobs:
           xl_label: 'size/xl'
           fail_if_xl: 'false'
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          files_to_ignore: >
-            package-lock.json
-            yarn.lock
-            go.sum
+          files_to_ignore: 'package-lock.json yarn.lock go.sum'
```

I still prefer the multi-line approach. The `>` tells it to join with spaces.
```suggestion
          files_to_ignore: >
            package-lock.json
            yarn.lock
            go.sum
```